packages/teams-js/src/public/meeting.ts

```diff
@@ -665,7 +723,7 @@ export namespace meeting {
 
     if (
       (response.details?.type == CallType.GroupCall || response.details?.type == CallType.OneOnOneCall) &&
-      !response.details.originalCaller
+      !response.details.originalCallerInfo
```

Comment:
There should be a unit test for this scenario (I assume there is not because if there was I would see an edited test below)

Reply:
Will add a test for this scenario, thanks

Reply:
Actually, it seems like I've included a test in the original PR:
https://github.com/OfficeDev/microsoft-teams-library-js/blob/753866fa9498d97f494ad2342a5a3c414ddbc157/packages/teams-js/test/public/meeting.spec.ts#L684C11-L727C14

The reason why this test is compatible even with this change is because it checks to see if the new property, `originalCaller` / `originalCallerInfo` is missing. The logic is that this new API should return the new property if it is supported, hence if the property isn't returned, the platform isn't supported.